src/components/map/controls/share/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { FC, useState, MouseEvent } from 'react';
+
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```

Comment:
Before adding this lib, did you consider if you can do it yourself? I think is small enough to replicate what you need in your code.
https://github.com/nkbt/react-copy-to-clipboard/blob/master/src/Component.js